Contract Template/admin/initServer.sqf

```diff
@@ -0,0 +1,14 @@
+// Do not Edit this file.
+
+// Log given traits to RPT 70 minutes after mission start.
+[
+    {
+        private _playerTraits = [];
+        {
+            _playerTraits pushBack [_x, _y];
```

Comment:
What is `_y`?

Reply:
`_x` is key `_y` is corresponding value [see BIKI](https://community.bistudio.com/wiki/HashMap#Iterating_through_a_HashMap)

Reply:
Oh interesting.